### ISSUE TITLE
Need guidance to implement multi tenancy

### ISSUE THREAD

**Issue by Jeyakumarrathnaselvi** [mirumee/saleor#1912]
We are new to Saleor and planning to implement multi tenancy with saleor for our requirement. Need different dashboard and storefront for each customers. 

Looks like we don't have this feature in Saleor

We need guidance whether we can implement it using?

http://django-tenant-schemas.readthedocs.io/en/latest/install.html

**Comment by patrys** [comment]
You should be able to use PostgreSQL schemas but this means that no information will be shared between the stores.

**Comment by jgtorrejon** [comment]
Yeah, got it, but Saleor provide any option about this feature..? or we need to implement this..?

**Comment by JsseL** [comment]
@Jeyakumarrathnaselvi Did you manage to use Saleor with django-tenant-schemas? Can you share any details about how it worked out?

**Comment by pgoreutn** [comment]
any news regarding multi tenant ????

**Comment by samuelviveiros** [comment]
First of all thank you for this wonderful platform. I can't even imagine the amount of effort you've put into creating and maintaining this project.

OK, so I'm new to Saleor, and I saw that there is a Channels feature. I saw that it has some similarities with the multi-tenant concept, and although the data and settings are not separated at the database level (PostgreSQL schemas), nor separation at the domain name level (as is done in django -tenants), I think that will work for my purposes for now, which is to separate the data for each "tenant".
